src/components/Cards/CreateWikiWizardStepThree.vue

```diff
@@ -54,32 +54,21 @@
             </template>
           </v-radio>
         </v-radio-group>
-
-        <h3 v-if="showTerms" class="mt-6">Terms of Use</h3>
-        <v-checkbox
-          v-model="value.terms"
-          :disabled="inFlight"
-          :rules="[() => !!value.terms || 'You must accept the Terms of Service.']"
-          v-if="showTerms"
-        >
-          <template v-slot:label>
-            <div>
-              I agree to the
-              <v-tooltip bottom>
-                <template v-slot:activator="{ on }">
-                  <a
-                    target="_blank"
-                    href="/terms-of-use"
-                    @click.stop
-                    v-on="on"
-                  >
-                    Terms of Use</a>
-                </template>
-                Opens in new window
-              </v-tooltip>.
-            </div>
-          </template>
-        </v-checkbox>
+        <div>
+          Previously accepted
+          <v-tooltip bottom>
+            <template v-slot:activator="{ on }">
+              <a
+                target="_blank"
+                href="/terms-of-use"
+                @click.stop
+                v-on="on"
+              >
+                Terms of Use</a>
+            </template>
+            Opens in new window
+          </v-tooltip> still apply.
+        </div>
```

Comment:
The text here is smaller than the rest of the form and Anton's mockup. A `text-body-1` class on the `div` should do the trick. However, is the text intentionally smaller because it doesn't fit on a single line otherwise? 

Let's ask UX if we should use a smaller font size or can increase the [max-width of this card](https://github.com/wbstack/ui/blob/T404988/src/components/Pages/CreateWiki.vue#L29) to 400px (technically 395px is the minimum it has to be for the text to fit).

Reply:
Yes it is. The conclusion was to keep the font smaller. UX prefered the text smaller as long as it fits in one line.

Reply:
cool, I'll click resolve :)